lib/charms/opensearch/v0/helper_conf_setter.py

```diff
@@ -289,10 +288,9 @@ def replace(
         if output_type in [OutputType.console, OutputType.all]:
             logger.info(data)
 
-        if output_file is None:
-            output_file = config_file
```

Comment:
this is deleted without any substitute - we should put back

Reply:
Yes, check my comment before